concent_api/core/payments/backends/sci_backend.py

```diff
@@ -55,6 +57,46 @@ def get_list_of_payments(
     return payments_list
 
 
+def are_all_payment_ts_younger_than_last_payment_closure_time_if_payment_exists(
```

Comment:
This name should be changed as well:
`is_last_closure_time_older_than_oldest_payment` and if yes we should reject. It seems simpler that way, at least to me

Reply:
done, it is validation now so I called it `validate_that_last_closure_time_is_older_than_oldest_payment`